builders/pallets-precompiles/precompiles/proxy.md

```diff
@@ -0,0 +1,124 @@
+
+ - **Any** — the any proxy will allow the proxy account to make any type of transaction
+ - **NonTransfer** — the non-transfer proxy will allow the proxy account to make any type of transaction, except for balance transfers
+ - **Governance** - the governance proxy will allow the proxy account to make any type of governance related transaction (includes both democracy or council pallets)
+ - **Staking** - the staking proxy will allow the proxy account to make staking related transactions
+ - **CancelProxy** - the cancel proxy will allow the proxy account to reject and remove delayed proxy announcements (of the primary account)
+ - **Balances** - the balances proxy will allow the proxy account to only make balance transfers
+ - **AuthorMapping** - this type of proxy account is used by collators to migrate services from one server to another
+ - **IdentityJudgement** - the identity judgement proxy will allow the proxy account to judge and certify the personal information associated with accounts on Polkadot
+
+## Interact with the Solidity Interface {: #interact-with-the-solidity-interface }
+
+### Checking Prerequisites {: #checking-prerequisites } 
+
+The below example is demonstrated on Moonbase Alpha, however, similar steps can be taken for Moonbeam and Moonriver. You should:  
+
+ - Have MetaMask installed and [connected to Moonbase Alpha](/tokens/connect/metamask/){target=_blank}
+ - Have an account with some DEV tokens.
+ - Have a second account that you control to use as a proxy account (funding optional).
+ --8<-- 'text/faucet/faucet-list-item.md'
+
+### Remix Set Up {: #remix-set-up } 
+
+1. Get a copy of [`Proxy.sol`](https://github.com/PureStake/moonbeam/blob/master/precompiles/proxy/Proxy.sol){target=_blank}
+2. Copy and paste the file contents into a [Remix file](https://remix.ethereum.org/){target=_blank} named `Proxy.sol`
+
+![Copying and Pasting the Proxy Interface into Remix](/images/builders/pallets-precompiles/precompiles/proxy/proxy-1.png)
+
+### Compile the Contract {: #compile-the-contract } 
+
+1. Click on the **Compile** tab, second from top
+2. Then to compile the interface, click on **Compile Proxy.sol**
+
+![Compiling Proxy.sol](/images/builders/pallets-precompiles/precompiles/proxy/proxy-2.png)
+
+### Access the Contract {: #access-the-contract } 
+
+1. Click on the **Deploy and Run** tab, directly below the **Compile** tab in Remix. Note: you are not deploying a contract here, instead you are accessing a precompiled contract that is already deployed
+2. Make sure **Injected Web3** is selected in the **ENVIRONMENT** drop down
+3. Ensure **Proxy.sol** is selected in the **CONTRACT** dropdown. Since this is a precompiled contract there is no need to deploy, instead you are going to provide the address of the precompile in the **At Address** Field
```

Comment:
```suggestion
3. Ensure **Proxy.sol** is selected in the **CONTRACT** dropdown. Since this is a precompiled contract there is no need to deploy, instead you are going to provide the address of the precompile in the **At Address** field
```